src/plugins/es_ui_shared/common/index.ts

```diff
@@ -6,4 +6,4 @@
  * Side Public License, v 1.
  */
 
-export { Privileges, MissingPrivileges } from '../__packages_do_not_import__/authorization';
+export { Privileges, MissingPrivileges } from '../__packages_do_not_import__/authorization/types';
```

Comment:
I had to change these types to be exported directly from the types file, otherwise it doesn't tree-shake the rest of the things exported from the `authorization` file and it causes errors if you try to use it server side (certain dependencies needed by the components exported require window object to exist).